libnavigation-core/src/main/java/com/mapbox/navigation/core/accounts/BillingController.kt

```diff
@@ -215,18 +217,23 @@ internal class BillingController(
                 is NavigationSessionState.Idle -> {
                     getRunningOrPausedSessionSkuId()?.let {
                         billingService.pauseBillingSession(it)
+                        logI(BILLING_EXPLANATION_CATEGORY) {
```

Comment:
@VysotskiVadim I think we can test it, and check that we print correct reason to logs.

Reply:
sure, we can! 
Let's not announce this feature in public documentation unit tests are in place.